docs/make.jl

```diff
@@ -1,6 +1,3 @@
-pushfirst!(LOAD_PATH, joinpath(@__DIR__, "..")) # add Enzyme to environment stack
-pushfirst!(LOAD_PATH, joinpath(@__DIR__, "..", "lib")) # add EnzymeCore to environment stack
-
```

Comment:
I can't say how much frustration these two lines caused me, I was getting all the time, on different machines, absurd errors about `LLVM` or `StaticArrays` not being installed, while they were clearly in my environment. This drove me crazy!